src/components/homepage/HomeFooter.jsx

```diff
@@ -87,7 +87,7 @@ function Safety({ className }) {
       )}
     >
       <div className="flex flex-1 place-items-center justify-center rounded-2xl bg-white px-4 py-6 font-jakarta font-bold text-gray-500">
-        Your Safety is utmost important to us
+        Your Security, Our Priority.
```

Comment:
@roerohan can you add a `<br>` between `Your Security,` & `Our Priority`

Reply:
Done